src/simperium/http-request.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import { request } from 'https'
+
+export default function( endpoint, body, options ) {
```

Comment:
Since `endpoint` isn't used in  would it make sense to get rid of `options` and move the `url.parse` here, then the arguments could be something like `key, url, body` and `url.parse` becomes an internal concern of how the non-browser version works?

That would allow `src/simperium/auth.js` to lose a whole import and simplify `getUrlOptions` a little bit.

Reply:
Flow is unhappy with this file (hence the CI failures), this does the trick for me:

```suggestion
export default function( endpoint: string, body: ( string|Buffer ), options: https$requestOptions ): Promise<string> {

```

Reply:
> would it make sense to get rid of options and move the url.parse here

yeah it would. I was starting with the smallest change I could think of. I'll update though. I didn't like passing `options` anyway